dojoWordle/components/Game/Game.style.tsx

```diff
@@ -1,8 +1,41 @@
 import styled from "@emotion/native";
+import { LetterBox, letterStatus } from "../LetterBox.tsx/LetterBox";
+
+export const wordToFind = ["b", "o", "n", "j", "o", "u", "r"];
 
 export const ResultContainer = styled.View({
   alignItems: "center",
   flexDirection: "row",
   justifyContent: "center",
   paddingBottom: 50,
 });
+
+export const displayCheckedWord = (wordGuessed: string) => {
+  var wordWithStatus = [];
+  var i: number;
+
+  {
+    for (i = 0; i < wordToFind.length; i++) {
+      var status: letterStatus;
+      status = "failure";
+
+      if (wordToFind[i] === wordGuessed[i]) {
+        status = "success";
+      } else if (wordToFind.find((character) => character === wordGuessed[i])) {
+        status = "missed";
+      }
+      wordWithStatus[i] = <LetterBox status={status} letter={wordGuessed[i]} />;
+    }
+  }
+  return wordWithStatus;
+};
+
+export const displayTypingWord = (wordTyped: string) => {
+  var wordBox = [];
+  var i: number;
+
+  for (i = 0; i < wordToFind.length; i++) {
```

Comment:
Can you refactor this for with Array.prototype.map?

You will see more often maps on projects. And it's a good exercise!